tests/core/conversations_worker/integration/__init__.py

```
@@ -0,0 +1,341 @@
+        _api_key=decoded["api_key"],
+        use_pgchanges=use_pgchanges,
+    )
+    yield fx
+
+    # Best-effort teardown: stop any still-active conversations and delete them
+    for cid in fx._created_conversations:
+        try:
+            conv = fx.get_conversation(cid)
+            if conv["status"] in ("pending", "queued", "running"):
+                fx.stop_conversation(cid)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)